content/release-10-17-0/datahub-10-17-0-bundle/10_17_0.md

```diff
@@ -22,7 +22,7 @@ The look and feel of the UI has been updated in line with the Cumulocity IoT pla
 |If the collection to be offloaded has JSON attributes consisting of more than 32,000 characters, its data cannot be offloaded. One specific case where this limitation applies is Cumulocity IoT's application builder, which stores its assets in the inventory collection when being used.|
 |If the collection to be offloaded has more than 800 JSON attributes, its data cannot be offloaded. This limitation also includes nested JSON content, which will be expanded into columns during offloading. Therefore, measurements documents with more than 800 series/series value fragments are not supported.|
 |If an attribute of a collection has varying types associated, the result table will contain a mixed type which may render query writing difficult or lead to problems with subsequent consumer applications.|
-|Dremio has announced in its [Dremio 4.0 release notes](https://docs.dremio.com/release-notes/40-release-notes.html#deprecations) to deprecate some functionality on mixed types. Since Dremio 13.2, GROUP BY or DISTINCT over mixed type expressions are no longer supported. It is necessary to cast these expressions to a simple type before using them in a GROUP BY or DISTINCT clause. Also mixed type expressions cannot be used in the SELECT clause, when retrieving data over ODBC or JDBC.|
+|Dremio has announced in its [Dremio 4.0 release notes](https://docs.dremio.com/release-notes/40-release-notes.html#deprecations) **broken link** to deprecate some functionality on mixed types. Since Dremio 13.2, GROUP BY or DISTINCT over mixed type expressions are no longer supported. It is necessary to cast these expressions to a simple type before using them in a GROUP BY or DISTINCT clause. Also mixed type expressions cannot be used in the SELECT clause, when retrieving data over ODBC or JDBC.|
```

Comment:
@BeateRixen this link doesn't work currently and I could not really find a new one on the Dremio docs website where it talks about this specifically.

Reply:
@cheinz-sag Can you please check?

Reply:
Hi,

great that you caught this one! Can we do it like that:
- for version 10.13 and above remove the complete entry "Dremio has announced..."
- for version 10.11 drop the link and say: Dremio has deprecated some functionality on mixed types.

If you're good with this, I'll set up corresponding branches.

Reply:
Sure. @dianaolives can you please prepare the PRs accordingly.

Reply:
@BeateRixen @cheinz-sag I've dropped the link for 10.11 and removed the whole row on the table for 10.13 and above.

Reply:
Hi Diana,

thanks! That looks good.